### PR TITLE
feat(cni): erctrl add proxy debug cmd

### PR DESCRIPTION
**Pull request by qingwusunny**

### CODE REVIEW

**Review by KeyongSun — APPROVED** [review]

File: cmd/everoute-agent/main.go [hunk 1]
```diff
@@ -82,20 +82,21 @@ func main() {
 		initCNI(datapathManager, mgr, proxySyncChan)
 		ovsdbMonitor = initOvsdbMonitor(datapathManager, stopChan)
 	} else {
-		// in vi
+		// in virtual senary, k8sCtrl manager initializer reply on ovsdbmonitor initialization to connect to kube-apiserver
```
Comment (wu-cl) [inline cmd/everoute-agent/main.go:85]: `senary` typo？

File: pkg/agent/controller/proxy/controller.go [hunk 2]
```diff
@@ -25,22 +25,54 @@ import (
...
+		backendObjs, _ := c.backendCache.ByIndex(proxycache.ServicePortIndex, svc.SvcID+"/"+portName)
+		for i := range backendObjs {
+			backends = append(backends, *backendObjs[i].(*proxycache.Backend).DeepCopy())
+		}
+	}
+
+	return svc, backends, svcPortRscNames
+}
+
 // Reconcile watch Service related resource and implement Service
```
Comment (wu-cl) [inline pkg/agent/controller/proxy/controller.go:58]: same with var name

File: pkg/agent/controller/proxy/controller.go [hunk 3]
```diff
@@ -25,22 +25,54 @@ import (
...
-	svcPortCache cache.Indexer
-	backendCache cache.Indexer
+	Cache
+}
+
+func (r *Reconciler) GetCache() *Cache {
+	return &r.Cache
 }
 
 // ReconcileService receive Service from work queue
```
Comment (wu-cl) [inline pkg/agent/controller/proxy/controller.go:74]: ditto

File: pkg/erctl/flowclient.go [hunk 4]
```diff
@@ -17,8 +17,13 @@ import (
...
 )
 
+const (
+	FlowType  = "flow"
+	GroupType = "group"
+)
+
 var (
-	bridgeNameSuffix = []string{"", "-policy", "-cls", "-uplink"}
+	bridgeNameSuffix = []string{"", "-policy", "-cls", "-uplink", "-nat"}
```
Comment (wu-cl) [inline pkg/erctl/flowclient.go:26]: if nat bridge is not exist, we might get nil in GetFlows?

**Review by wu-cl — APPROVED** [review]

**Review by KeyongSun — APPROVED** [review]

**Pull request merged**